nextcloud_update.sh

```diff
@@ -395,6 +395,15 @@ fi
 
 countdown "Backing up files and upgrading to Nextcloud $NCVERSION in 10 seconds... Press CTRL+C to abort." "10"
 
+# Backup app status
+print_text_in_color "$ICyan" "Getting and backing up the status of apps for later. This can take a while..."
+NC_APPS="$(occ_command app:list | sed '/Enabled/d' | sed '/Disabled/d' | sed 's|^  - ||g' | sed 's|:.*||')"
```

Comment:
That `sed` looks very "seddy". :)

Jokes aside, use this instead: `app:list | sed '/Enabled/d' | sed '/Disabled/d' | cut -d "-" -f2 | cut -d ":" -f1 | tr -d " "`

Reply:
This is even cleaner: `app:list | awk '{print$2}' | tr -d ":" | sed '/^\s*$/d'`

Reply:
And even more clean: `app:list | awk '{print$2}' | tr -d ":" | sed '/^$/d'´

Reply:
I will try out your suggestion later when I am back home.

Reply:
`awk` prints the second "thing", `tr` removes the : and `sed` removes the whitespace.

You could use `cut` instead of `tr` for readability as well. Your choice.

Reply:
Prune for bugs if the output changes though. Maybe ad a check for that?

Reply:
I will see if a check is possible

Reply:
A simple one could be:

```
if [ -z "$NC_APPS" ]
then
    print_text_in_color "$IRed" "No apps detected, aborting export of settings... Please report this issue to $ISSUES"

------SKIP EXPORT
fi

Reply:
> And even more clean: `app:list | awk '{print$2}' | tr -d ":" | sed '/^$/d'`

This has worked in my testing and was addressed.

Reply:
> A simple one could be:
> 
> ```
> if [ -z "$NC_APPS" ]
> then
>     print_text_in_color "$IRed" "No apps detected, aborting export of settings... Please report this issue to $ISSUES"
> 
> ------SKIP EXPORT
> $APPSTORAGE=no-export-done
> fi
> ```
> 
> Then with the import:
> 
> ```
> if [ $APPSTORAGE = no-export-done ]
> then
> .................... DON'T IMPORT
> fi
> ```

this was addressed in the latest commit